requirements.txt

```diff
@@ -24,4 +24,4 @@ pdfkit>=0.6.1
 cryptography>=3.3.2
 httplib2>=0.19.0
 tritonclient[all]>=2.4.0
-gevent>=21.12.0
+gevent>=21.12.0
```

Comment:
is anything different here? if not, revert

Reply:
Reverted